pkg/crc/constants/constants_darwin.go

```diff
@@ -11,6 +11,7 @@ const (
 	PodmanRemoteExecutableName = "podman"
 	TrayExecutableName         = "Red Hat OpenShift Local.app"
 	DaemonAgentLabel           = "com.redhat.crc.daemon"
+	QemuGAPort                 = 1234
```

Comment:
Can't we just call it `QemuGuestAgentPort` ?

Reply:
I changed it.